extension/parquet/parquet_reader.cpp

```diff
@@ -1104,14 +1104,15 @@ bool ParquetReader::ScanInternal(ClientContext &context, ParquetReaderScanState
 			}
 			auto &scan_filter = state.scan_filters[state.adaptive_filter->permutation[i]];
 			auto filter_entry = reader_data.filter_map[scan_filter.filter_idx];
-			if (filter_entry.is_constant) {
+			if (filter_entry == DConstants::INVALID_INDEX) {
 				// this is a constant vector, look for the constant
-				auto &constant = reader_data.constant_map[filter_entry.index].value;
+				D_ASSERT(reader_data.constant_map.count(scan_filter.filter_idx));
+				auto &constant = reader_data.constant_map[scan_filter.filter_idx];
```

Comment:
This is the only place that could be negatively impacted by the change from vector to unordered_map I feel, as we previously directly indexed into an array, and now we do a hash and lookup in the unordered map